focus-areas/contributions/types-of-contributions.md

```diff
@@ -1,82 +1,67 @@
+Additionally, types of contributions along with project demographic information may provide insight into the diversity and inclusiveness of a project, helping inform decisions around project governance and how new contributors are welcomed.
+
+## **Want to Know More?**
+
+<span markdown="1"><details>
+<summary>Click to read more about this metric.</summary>
+
+### **Data Collection Strategies**
+- **Interviews or Surveys:** Ask community members to recognize others for their contributions to identify contribution types that may have been overlooked. Example questions:
+   - *Who in the project would you like to recognize for their contributions? What did they contribute?*
+- **Observations:** Identify and recognize leaders in different parts of the project.
+   - *What leaders are listed on the project website or in a repository?*
+- **Non-code Contributions Tracking:** Use a dedicated system, e.g., an issue tracker, to log non-code contributions.
+   - For example, communication channel activity like mailing list discussions can be a proxy for QA contributions.
+- **Trace Data Collection:** Measure contributions through collaboration tool logs (e.g., repository commits, wiki edits, or email archives).
+- **Automated Classification:** Train an AI bot to detect and classify contributions (e.g., distinguishing help requests from support provided).
+
+### **Filters**
+- Contribution types might include:
+  - Writing Code
+  - Reviewing Code
+  - Bug Triaging
+  - Quality Assurance and Testing
+  - Security-Related Activities
+  - Localization and Translation
+  - Event Organization
+  - Documentation Authorship
+  - Community Building and Management
+  - Social Media Management
+  - Marketing and Campaign Advocacy
+
+### **Visualizations**
+- None Specified
+
+</details></span>
+
+## **References**
+- [Revisiting the Word Recognition in FOSS and the Dream of Open Credentials](https://medium.com/@sunnydeveloper/revisiting-the-word-recognition-in-foss-and-the-dream-of-open-credentials-d15385d49447)
+- [14 Ways to Contribute to Open Source Without Being a Coding Expert](https://smartbear.com/blog/test-and-monitor/14-ways-to-contribute-to-open-source-without-being/)
+- [A Guide to Issue Credits and the Drupal.org Marketplace](https://www.drupal.org/drupalorg/blog/a-guide-to-issue-credits-and-the-drupal.org-marketplace)
+- [OpenStack AUC Recognition](https://wiki.openstack.org/wiki/AUCRecognition)
+- [Contributing](https://24pullrequests.com/contributing)
+
+## **Contributors**
+- Kevin Lumbard
+- Vinod Ahuja
+- Elizabeth Barron
+- Dawn Foster
+- Sean Goggins
+- Matt Germonprez
+- Yigakpoa L. Samuel (Ikpae)
+
+## **Additional Information**
+- To edit this metric please submit a Change Request here: https://github.com/chaoss/wg-common/blob/main/focus-areas/contributions/types-of-contributions.md
```

Comment:
Can you change the path to match the path of this file? (We changed the name of this repo from wg-common to wg-metrics-development a while back)

Reply:
Hi Elizabeth, thank you for pointing this out. It’s done now